addons/l10n_uy/demo/account_demo.py

```diff
@@ -1,20 +1,490 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import api, models
+import logging
+from odoo import api, models, Command
+import time
+
+from odoo.exceptions import UserError, ValidationError
+_logger = logging.getLogger(__name__)
 
 
 class AccountChartTemplate(models.AbstractModel):
 
     _inherit = "account.chart.template"
 
+    @api.model
+    def _get_demo_data(self, company=False):
+        return {
+            'account.move': self._get_demo_data_move(company),
+            'account.move.reversal': self._get_demo_data_move_reversal(company),
+            'res.partner': self._get_demo_data_res_partner(company),
+        }
+
+    def _post_load_demo_data(self, company=False):
+
+        invoices = (
+            self.ref('demo_invoice_1')
+            + self.ref('demo_invoice_2')
+            + self.ref('demo_invoice_3')
+            + self.ref('demo_invoice_4')
+            + self.ref('demo_invoice_5')
+            + self.ref('demo_invoice_6')
+            + self.ref('demo_invoice_8')
+            + self.ref('demo_invoice_9')
+            + self.ref('demo_sup_invoice_1')
+            + self.ref('demo_sup_invoice_2')
+            + self.ref('demo_sup_invoice_3')
+            + self.ref('demo_sup_invoice_6')
+            + self.ref('demo_sup_invoice_7')
+            + self.ref('demo_sup_invoice_8')
+            + self.ref('demo_sup_invoice_9')
+        )
+        # the invoice_extract acts like a placeholder for the OCR to be ran and doesn't contain
+        # any lines yet
+        for move in invoices:
+            try:
+                move.action_post()
+            except (UserError, ValidationError):
+                _logger.exception('Error while posting invoices')
+
+        # Post the reversal moves
+        reversal_moves = []
+        invoices_to_revert = (
+            self.ref('demo_refund_invoice_1')
+            + self.ref('demo_refund_invoice_2')
+            + self.ref('demo_refund_invoice_3')
+            + self.ref('demo_refund_invoice_4')
+            + self.ref('demo_sup_refund_invoice_3')
+            + self.ref('demo_sup_refund_invoice_2')
+            )
+        for move in invoices_to_revert:
+            reversal_moves.append(move.refund_moves().get('res_id'))
+        for move in self.env['account.move'].search([('id', 'in', reversal_moves)]):
+            try:
+                move.action_post()
```

Comment:
It seems a bit weird to loop twice, and you could use a browse instead of the search.
For example, something like this seems better to me
```suggestion
            try:
                reversal_move = self.env['account.move'].browse(move.refund_moves().get('res_id'))
                reversal_move.action_post()
```
You can also separate in 3 lines if you prefer to not do the browse + refund_moves in the same line.